crates/sage-cli/tests/integration.rs

```diff
@@ -8,16 +8,17 @@ use sage_core::spectrum::SpectrumProcessor;
 #[test]
 fn integration() -> anyhow::Result<()> {
     let mut builder = Builder::default();
-    builder.update_fasta("../../tests/Q99536.fasta".into());
+    builder.update_fasta("foo".into());
```

Comment:
Should try to remove the `fasta` field from parameters while maintaining backwards compatibility